pkg/reconciler/revision/queueing_test.go

```diff
@@ -61,7 +63,7 @@ const (
 	testQueueImage      = "queueImage"
 )
 
-func testRevision() *v1alpha1.Revision {
+func testRevision(classAnnotationValue ...string) *v1alpha1.Revision {
```

Comment:
Is this a variadic argument only to be able to not change existing occurrences? That's kinda confusing.

Let's just add a `testRevisionWithClass` or something that does the "right thing" and only change where necessary. `testRevision` should call `testRevisionWithClass` with an empty string then.